src/librustc_save_analysis/lib.rs

```diff
@@ -110,6 +112,24 @@ impl<'l, 'tcx: 'l> SaveContext<'l, 'tcx> {
         }
     }
 
+    // Returns path to the compilation output (e.g. libfoo-12345678.rmeta)
+    pub fn compilation_output(&self, crate_name: &str) -> PathBuf {
```

Comment:
I thought you said you were removing the output stuff? (Which seems like a good idea to me).

Reply:
So this doesn't use the file-based approach to figure out dependencies (crate disambiguators are used instead) but if we decide to somehow use save-analysis to lower this info further into low level file-based dep graph (potentially for Cargo build system integration? or maybe we shouldn't use save-analysis for that?) then we need some kind of related file input/output.

I removed `CrateSource` because one-of-dylib-rlib-rmeta-is-some format is clunky to work with but left this for the reasons explained above. Should I remove this as well?